test/unit/tasks/seeds_test.rb

```diff
@@ -15,6 +15,7 @@ class SeedsTest < ActiveSupport::TestCase
     Setting.stubs(:[]).with(:bcrypt_cost).returns(1)
     Setting.stubs(:[]=).with(:bcrypt_cost, anything).returns(true)
```

Comment:
we will probably want to stub the new settings as well, right?

Reply:
This is only for seed test, no need to stub unused settings I guess.

Reply:
Whatever I can add them all just to be sure to prevent issues.